lib/tags/collect_tag_resources.rb

```diff
@@ -21,7 +21,7 @@ def local_tag_resources
     end
 
     def remote_tag_resources
-      @remote_tag_resources = Topology::RemoteInventory.new(@task).process.tag_resources
+      @remote_tag_resources = Inventory::RemoteInventory.new(@task).process.tag_resources
```

Comment:
@hsong-rh Should this be CatalogInventory instead of Inventory

Reply:
changed